collectors/cveorg/keywords.py

```diff
@@ -1,786 +1,32 @@
-    "Unitrends Backup",
-    "Untis WebUntis",
-    "Userscape HelpSpot",
-    r"VIDEOJET.*[\n]*.*psirt",
-    "VMware ESXi and vCenter Server",
-    "VMware Fusion",
-    "VMware Workstation",
-    "Vehicle Booking System",
-    "Verint Workforce Optimization",
-    "Veritas NetBackup",
-    "Verizon 5G Home",
-    "Vesta Control Panel",
-    "Victor CMS",
-    "VirtueMart",
-    "WBCE CMS",
-    "WECON LeviStudioU",
-    "WPS Office",
-    "WSO2 Enterprise Integrator",
-    "WTCMS",
-    "WUZHI CMS",
-    "WatchDog Anti-Malware",
-    "Wavlink",
-    "Web Based Quiz System",
-    "WebDynpro Java",
-    "Weblication CMS",
-    "Wedding Management System",
-    "Wedding Planner",
-    "Weeny Audio Cutter",
-    "Wellcms",
-    "Western Bridge Cobub Razor",
-    "Western Digital My Cloud",
-    "Winmail",
-    "Wireless IP Camera 360",
-    "WoWonder",
-    "WonderCMS",
-    "WordPress theme",
-    r"WordPress.*plugin",
-    "Wowza Streaming",
-    "XYHCMS",
-    "Xiaomi.*phones",
-    "Xiuno BBS",
-    "XunRuiCMS",
-    "Yab Quarx",
-    "Yahoo!",
-    "Yxcms",
-    "YxtCMF",
-    "YzmCMS",
-    "Z-BlogPHP",
-    "Zenario CMS",
-    "Zikula Application Framework",
-    "Zoho ManageEngine",
-    "ZoneAlarm",
-    "ZoneMinder",
-    "Zoo Management System",
-    "Zulip Desktop",
-    "Zyxel",
-    "baijiacms",
-    "bootstrap-table",
-    "chatwoot",
-    "cmseasy",
-    "comforte SWAP",
-    "concretecms",
-    "dotCMS",
-    "drawio",
-    "eDNA Enterprise Data Historian",
-    "ebCMS",
-    r"ednareporting\.asmx",
-    "elitecms",
-    "emoncms",
-    "enhavo CMS",
-    "htmly",
-    "https://gitee.com/oufu/ofcms/",
-    "https://github.com/cesanta/mjs/",
-    "https://github.com/kabirkhyrul/HMS/",
-    "https://github.com/vapor/vapor/",
-    "https://github.com/wp-plugins",
-    "https://support.zte.com.cn/support/",
-    "https://www.autodesk.com/",
-    "https://www.solarwinds.com/",
-    "iDashboards",
-    "iPayPal",
-    "iRedMail",
-    "iScripts SupportDesk",
-    "iScripts UberforX",
-    "iScripts eSwap",
-    "iTunes",
-    "iota All-In-One Security Kit",
-    "ismartgate PRO",
-    "joyplus-cms",
-    "lyadmin",
-    "madlib-object-utils",
-    r"mySCADA myPRO|Measuresoft ScadaPro",
-    "open5gs",
-    "perfex crm",
-    "phpjs",
-    "pimcore",
-    r"plugin <= [0-9\.]+ at WordPress",
-    r"plugins.*wordpress",
-    "portfolioCMS",
-    "prime-jwt",
-    "publify",
-    "puppyCMS",
-    "rap2hpoutre Laravel Log Viewer",
-    "rdiffweb",
-    r"siteserver (CMS|SSCMS)",
-    "swftools",
-    "totaljs",
-    "trudesk",
-    "usememos/memos",
-    "vBulletin",
-    r"win32k\.sys",
-    "wityCMS",
-    "wuzhicms",
-    "yetiforcecrm",
-    "zzcms",
-]
+logger = get_task_logger(__name__)
 
-BLOCKLIST_CASE_SENSITIVE = ["iOS"]
 
-KEYWORD_ALLOWLIST = [
-    re.compile(rf"\b{keyword}\b", re.IGNORECASE) for keyword in ALLOWLIST
-] + [re.compile(keyword) for keyword in ALLOWLIST_SPECIAL_CASES]
+def fetch_keywords_from_ps_constants():
```

Comment:
I don't think this should be here. PS Constants should be fetched via the PS Constants collector, saved as a small model of the needed info and used from there instead of doing the fetch every time it is needed. There are already couple of examples https://github.com/RedHatProductSecurity/osidb/blob/master/collectors/ps_constants/core.py (eg. special consideration packages, sla policies, etc.)